cmd/cli/cmd/migrate.go

```diff
@@ -37,7 +37,7 @@ func NewMigrate() *cobra.Command {
 		Supported Botkube bot platforms for migration:
 		- Socket Slack
 		- Discord
-		- Mattermost
+		- Mattermost 
```

Comment:
Please remove empty line
```suggestion
		- Mattermost
```

and run `make gen-docs-cli`. Thanks!